src/lexedata/edit/add_central_concepts.py

```diff
@@ -212,6 +216,7 @@ def add_central_concepts_to_cognateset_table(
         if not overwrite_existing and row[c_core_concept]:
             continue
         row[c_core_concept] = central.get(row[dataset.column_names.cognatesets.id])
+        row["Status_Column"] = status_update
```

Comment:
This can only happen if the status column exists, so it needs an `if` or a `try`.

Reply:
I was wrong there. Because `pycldf` is fine with just ignoring columns it doesn't know about (which is sometimes handy, like here, but more often lead to unexpected behaviour, so I'm not sure whether I will at some point lobby to change that behaviour…), this does work as intended. I will nonetheless guard it with an `if`.